app/src/main/res/values/colors.xml

```diff
@@ -10,13 +10,10 @@
     <color name="icons">#FFFFFF</color>
     <color name="divider">#B6B6B6</color>
 
-    <color name="build_state_passed">#FF3FA75F</color>
```

Comment:
I saw that the errored state was shown in red, but in gray. While I was adding the gray anyway, I also updated the other colors to match the current web interface.